appservice/src/intTest/groovy/nu/yona/server/ActivityTest.groovy

```diff
@@ -1545,6 +1546,61 @@ class ActivityTest extends AbstractAppServiceIntegrationTest
 		appService.deleteUser(richardDefault)
 	}
 
+	def 'Richard\'s network activity is measured on iOS only'(def operatingSystem, def expectedValue)
+	{
+		given:
+		User richard = addRichard(false, operatingSystem)
+		setCreationTimeOfMandatoryGoalsToNow(richard)
+		addTimeZoneGoal(richard, SOCIAL_ACT_CAT_URL, ["01:00-12:00"], "W-1 Mon 02:18")
+		richard = appService.reloadUser(richard, CommonAssertions.&assertUserGetResponseDetailsWithPrivateDataIgnoreDefaultDevice)
+		Goal timeZoneGoalSocialRichard = richard.findActiveGoal(SOCIAL_ACT_CAT_URL)
```

Comment:
I would like an explicit test as well that asserts that network activity on No-go goals is recorded for both iOS and Android. That way, we have tested the critical difference, and can omit the device aspect from other tests.

Reply:
See ``BasicBuddyTest.'Goal conflict of Bob (on iOS and Android) is reported to Richard and Bob'``.